pkg/ebpf/bpf_bpfeb.go

```diff
@@ -91,7 +90,8 @@ type BpfSpecs struct {
 //
 // It can be passed ebpf.CollectionSpec.Assign.
 type BpfProgramSpecs struct {
-	IngressNodeFirewallProcess *ebpf.ProgramSpec `ebpf:"ingress_node_firewall_process"`
+	TcxIngressNodeFirewall        *ebpf.ProgramSpec `ebpf:"tcx_ingress_node_firewall"`
+	XdpIngressNodeFirewallProcess *ebpf.ProgramSpec `ebpf:"xdp_ingress_node_firewall_process"`
```

Comment:
nit: why "Process" at the end of `XdpIng.." ? better to keep the naming between tcx and xdp the same?

Reply:
will do and keep the same naming